app/controllers/paws_up_controller.rb

```diff
@@ -13,8 +14,8 @@ def create
 
   def index
     if params[:secret].nil? || params[:secret] == ENV["secret"]
-      @current_paws_up = PawsUp.where('created_at > ?', 1.week.ago).order('created_at DESC')
-      @previous_paws_up = PawsUp.where('created_at < ?', 1.week.ago).order('created_at DESC')
+      @current_paws_up = PawsUp.where(cycle: Cycle.last).order('created_at DESC')
```

Comment:
If you don't want to write explicit sql in order, I believe you can do
`.order(created_at: :desc)` here and below.

Reply:
Sure! Kinda just used what was there before but that sounds like a good idea